apps/server/src/shared/domain/service/lesson-copy.service.ts

```diff
@@ -19,22 +19,56 @@ export class LessonCopyService {
 			hidden: true,
 			name: this.nameCopyService.deriveCopyName(params.originalLesson.name),
 			position: params.originalLesson.position,
+			contents: this.copyTextComponent(params.originalLesson.contents),
```

Comment:
```suggestion
			contents: this.copyTextComponent(params.originalLesson.contents || []),
```
to avoid strange bug, we can't trust db to always have a contents propriety. (Trying to set it default in the entity does not seem to work for me) 
otherwise, this has issues below, when we try to loop